cls/SourceControl/Git/Utils.cls

```diff
@@ -183,6 +183,14 @@ ClassMethod IsImportAfter(menuItemName As %String) As %Boolean [ CodeMode = expr
 $Find(..#ImportAfterGitMenuItems, ","_menuItemName_",") > 0
 }
 
+ClassMethod MigrateInstanceSettings()
+{
+    if '$data(^["%SYS"]SYS("SourceControl", "Git")) {
```

Comment:
Not quite there, the condition here is flipped. Make sure to test on an instance where ^["%SYS"]SYS("SourceControl","Git","%gitBinPath") is set.

Reply:
Oops - missed this in my quick read through. Good catch.